### DISCUSSION TITLE
Builder gets hanged when i select an element with more than 25 class names

### DISCUSSION THREAD

**Discussion by kamalesh-d** [GrapesJS/grapesjs#5351] [General]
Hello, 

I have been using Grapesjs for quite a long time, and it has been always a great tool. Thanks for this amazing tool!

I am developing a website builder for TailwindCSS framework. As everyone knows that TailwindCSS allows the user to style the element by using their utility classes, so sometimes to design a stylish responsive button, the user might need to use more than 25 classes. It would include hover state, change in font size based on device, focus states, and some times even dark mode. So when we select a button which has 10 class names or something similar, the builder works very fine. However, when we select an element, for instance button with more than 25 class names (required for styling the button), the editor gets hanged. I am not able to make any further changes. The browser indicates that page unresponsive. Please find the issue reproduced in below demo. 

https://jsfiddle.net/ve89m1a4/4/

In the above link, when you select the button of "Log In", the editor works fine, (indicated by - we can edit the text inside it). However, when you select the "Get Started" button in the above link, the editor gets hanged. We are not able to style them properly. 

I agree it is not common issue for many others, as they might be working with pure CSS. But for people using Tailwind CSS or Bootstrap kind of frameworks, they might need to use more classes like this, which might causes this issue for them too. 

Can you please check this and guide me the solution for this? If you need more information, please let me know. 

Thanks, 
Kamalesh D.

**Comment by ronaldohoch** [comment]
Hello, I came back here after a long time to test and apparently the editor is no longer hanging. Did you notice too?

**Reply by artf** [reply]
Yeah it was reported and fixed [here](https://github.com/GrapesJS/grapesjs/issues/5525)
